Site/SiteOlarkModule.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+require_once 'Site/SiteApplicationModule.php';
+
+/**
+ * Module for dislaying Olark live chat
+ *
+ * @package   Site
+ * @copyright 2015 silverorange
+ * @link      https://www.olark.com/api
+ */
+class SiteOlarkModule extends SiteApplicationModule
+{
+	// {{{ public properties
+
+	/**
+	 * Locale strings
+	 *
+	 * @var array
+	 * @see https://www.olark.com/help/languages
+	 */
+	public $locale_strings = array();
+
+	// }}}
+	// {{{ protected properties
+
+	/**
+	 * Olark identity
+	 *
+	 * @var string
+	 */
+	protected $identity;
```

Comment:
I think this should be changed to `identify` or `site_id`. The current name of `identity` doesn't have any meaning when searching through Olark documentation.